### ISSUE TITLE
Ability to view the time required for each deck separately

### ISSUE THREAD

**Issue by Benjamin-Watson0** [cjdduarte/MDS_Time_Left#23]
At present, I can only see the time required for all the current decks. I believe it will be more convenient.

Thank you for your selfless work.😊

**Comment by akavi1** [comment]
Agree, I would also like to see this feature. Additionally it would be nice to specify which deck's stats should be displayed on the main screen if you dont want to see an aggregate of all the decks

**Comment by mattgarbecki** [comment]
Agree! The time left remaining is quite inaccurate for me as I use parent & subdecks when studying, and this double counts cards in those decks. I'd like to specify which decks to include in the stats so I can only choose the parent decks. Thank you!
